CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Fixed
 * Fixed a fatal error (reported to the sync error handler) during client reset (or automatic PBS to FLX migration) if the reset has been triggered during an async open and the schema being applied has added new classes. ([#6601](https://github.com/realm/realm-core/issues/6601), since automatic client resets were introduced in v11.5.0)
+* Fixes several issues with full text search ([#6591](https://github.com/realm/realm-core/issues/6591), since v13.0.0)
```

Comment:
I think this should be more specific - a user should be able to tell whether an issue they were experiencing is addressed by this change or not.

Reply:
I decided just to describe the main issue.